superduper/base/logger.py

```diff
@@ -37,20 +37,12 @@ class Logging:
             }
         )
 
-        # fmt = (
-        #     "<green> {time:YYYY-MMM-DD HH:mm:ss.SS}</green>"
-        #     "| <level>{level: <8}</level> "
-        #     "| <cyan>{extra[hostname]: <8}</cyan>"
-        #     "| <cyan>{name}</cyan>:<cyan>{line: <4}</cyan> "
-        #     "| <level>{message}</level>"
-        # )
-
         fmt = (
-            "{time:YYYY-MMM-DD HH:mm:ss.SS}"
-            "| {level: <8} "
-            "| {extra[hostname]: <8}"
-            "| {name}:{line: <4} "
-            "| {message}"
+            "<green> {time:YYYY-MMM-DD HH:mm:ss.SS}</green>"
```

Comment:
This should be configurable. We had this before, and it made life difficult with our `ray` deployments.

Reply:
Hi @blythed.I update it. please review it. Thank you.